### PR TITLE
week2

### PR DESCRIPTION
**Pull request by husenjanobul**

### CODE REVIEW

File: Week2/HomeWork/app.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const capitalCity = `select name from city where id in (select capital from country where name = ?)`;
```
Comment (mogaleaf) [inline Week2/HomeWork/app.js]: Good

File: Week2/HomeWork/app.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+const regions = `select distinct language from countrylanguage where coutrycode in (select code from county where region = ? )`;
```
Comment (mogaleaf) [inline Week2/HomeWork/app.js]: Be careful you wrote county instead of country

File: Week2/HomeWork/app.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+const cities = `select count(distinct name) as 'total' from city where countrycode in (select from countrylanguage where language = ?)`;
```
Comment (mogaleaf) [inline Week2/HomeWork/app.js]: This query has a select problem. You did not write the table name

File: Week2/HomeWork/app.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+const officialLanguage = `select name from country where region = ?
```
Comment (mogaleaf) [inline Week2/HomeWork/app.js]: Good

File: Week2/HomeWork/app.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+const capitalCity = `select name from city where id in (select capital from country where name = ?)`;
+
+const regions = `select distinct language from countrylanguage where coutrycode in (select code from county where region = ? )`;
+
+const cities = `select count(distinct name) as 'total' from city where countrycode in (select from countrylanguage where language = ?)`;
+
+const officialLanguage = `select name from country where region = ? 
+    and code in(select countrycode from countrylanguage where language = ? and isofficial = 't')`;
+
+const languages = `select country.continent, count(distinct language) as 'total language' from country
```
Comment (mogaleaf) [inline Week2/HomeWork/app.js]: be careful you have used county instead of country again :)

Comment (husenjanobul) [reply Week2/HomeWork/app.js]: thank you for review and feedback .
issues are fixed :)

**Pull request closed without merging**